axelrod/tests/unit/test_fingerprint.py

```diff
@@ -11,6 +28,141 @@
     matplotlib_installed = False
 
 
+class TestFingerPrintHelpers(unittest.TestCase):
+
+    def test_create_points_progress_bar_true(self):
```

Comment:
new tests for create_points.  I had expected create_points(0.3) to return points in [0.0, 0.3, 0.9]  so i thought it would be good to explicitly state that is not so.